vgen_process_simulink_model.m

```diff
@@ -126,7 +126,7 @@
 
 %% Generate kernel module build files
 disp('Creating Makefile and Kbuild.')
-system("python3 " + mp.driver_codegen_path + filesep + "gen_makefile.py " + [hdlpath filesep] + " " + mp.model_name);
+system("python " + mp.driver_codegen_path + filesep + "gen_makefile.py " + [hdlpath filesep] + " " + mp.model_name);
```

Comment:
On newer versions of Ubuntu (and probably other Linux distros, but we really only "officially" support Ubuntu), there is no longer a symlink from python{2,3} -> python, so you have to call python3 explicitly on Ubuntu now. 

We will need an `ispc` and `isunix` check to call the appropriate python binary.

Reply:
Added